charts/aws-efs-csi-driver/values.yaml

```
@@ -104,6 +106,12 @@ controller:
     #  eks.amazonaws.com/role-arn: arn:aws:iam::111122223333:role/efs-csi-role
   healthPort: 9909
   regionalStsEndpoints: false
+  # Pod Disruption Budget
+  podDisruptionBudget: 
+    enabled: false
+    maxUnavailable: 1
+    minAvailable: 2
```

```suggestion
  podDisruptionBudget: 
    enabled: false
    # maxUnavailable: 1
    minAvailable: 1
```

Since default replica count is `2`, then we should make the PDB allow at-least one disruption. 
Also, you cannot define both `minAvailable` and `maxUnavailable` together. So we have to comment-out one (leave it as comment to indicate that PDB template support setting `maxUnavailable` too.